xdp/src/packet.rs

```diff
@@ -0,0 +1,116 @@
+#![allow(clippy::arithmetic_side_effects)]
+
+use {libc::ETH_P_IP, std::net::Ipv4Addr};
+
+pub const ETH_HEADER_SIZE: usize = 14;
+pub const IP_HEADER_SIZE: usize = 20;
+pub const UDP_HEADER_SIZE: usize = 8;
+
+pub fn write_eth_header(packet: &mut [u8], src_mac: &[u8; 6], dst_mac: &[u8; 6]) {
```

Comment:
These can probably land in net-utils as well (they can be benefitial for other bits of code not just XDP)

Reply:
See above